scripts/pi-hole/php/password.php

```diff
@@ -9,114 +9,100 @@
 
 require_once('func.php');
 
-// Start a new PHP session (or continue an existing one)
 // Prevents javascript XSS attacks aimed to steal the session ID
 ini_set('session.cookie_httponly', 1);
 // Prevent Session ID from being passed through URLs
 ini_set('session.use_only_cookies', 1);
+
+// Start a new PHP session (or continue an existing one)
 session_start();
 
 // Read setupVars.conf file
 $setupVars = parse_ini_file("/etc/pihole/setupVars.conf");
+
 // Try to read password hash from setupVars.conf
-if(isset($setupVars['WEBPASSWORD']))
-{
+if (isset($setupVars['WEBPASSWORD'])) {
     $pwhash = $setupVars['WEBPASSWORD'];
-}
-else
-{
+} else {
     $pwhash = "";
 }
```

Comment:
This looks like an unnecessary code-style change. Even more of this happens below. If we ever were to track down changes "when did what happen" in the future, such changes without any really change always make lives more difficult without any apparent gain.

Reply:
https://javascript.info/coding-style#curly-braces

Reply:
But really, styling changes should come in a separate PR/commit I suppose (though toggling "hide whitespace" makes it easer to review)

Reply:
But really, styling changes should come in a separate PR/commit I suppose

Reply:
Actually, the link you posted is about javascript style.

I understand the code style is a personal preference, but we need to choose and follow a single standard.
Currently there is no style for PHP files (or the style is a complete mess). I want to fix this using the same style to all PHP files.

I can suggest one of the know PHP styles ([PSR-2](https://www.php-fig.org/psr/psr-2/), [PSR-12](https://www.php-fig.org/psr/psr-12/)).

If you like, I can delay this PR an format all PHP files in a new PR.

Reply:
For Vs code there are (multiple) php-code style fixers (e.g. https://github.com/junstyle/vscode-php-cs-fixer).

We could also consider adding an action to check the code style: https://github.com/OskarStark/php-cs-fixer-ga which uses https://github.com/FriendsOfPHP/PHP-CS-Fixer internally.

Reply:
Yes, like we already use for other languages.

Reply:
There's some PHP modules for `prettier` too. I have most of my linting via `prettier`.

Reply:
I read somewhere prettier-PHP is not so good for files mixing HTML and PHP (our case), but we can try it.

Reply:
Yes, fixing mixed html/php is considered unstable: https://github.com/prettier/plugin-php

I don't see a way to specify `PSR-12` or `PSR-2` directly with this plugin.

Reply:
`prettier` doesn't allow select many options. It uses what `prettier` developers decided.

Reply:
`PSR-2` has been superseded by `PSR-12` according to [the specifications](https://www.php-fig.org/psr/psr-12/) you liked for `PSR-12`:

> This specification extends, ***expands and replaces [PSR-2](https://www.php-fig.org/psr/psr-2/)***, the coding style guide and requires adherence to [PSR-1](https://www.php-fig.org/psr/psr-1/), the basic coding standard.

Reply:
Yes, it includes PSR-2 + PSR-1 rules, but most new rules in `PSR-12` are related to classes (which we don't use in our code) and "new functionalities" like type declarations (`declare(strict_types=1)`).

**Note:**
Not really _new_ functionality, but from PHP7.4+.

We still support PHP5.6 and we don't use classes, so I think we can use PRS-2 style without any problem.

Reply:
@rdwebdesign I pushed a new branch using the linter suggested by @yubiuser above: [tweak/php-cs-fixer](https://github.com/pi-hole/AdminLTE/tree/tweak/php-cs-fixer)

I used `php-cs-fixer` locally to fix all files and will open a PR now with some further details.

Reply:
https://github.com/pi-hole/AdminLTE/pull/2288